src/app/Shared/Services/CommandChannel.service.tsx

```diff
@@ -35,17 +35,18 @@ export class CommandChannel {
 
     fromFetch(`${this.apiSvc.authority}/health`)
       .pipe(
-        concatMap(resp => from(resp.json())))
-      .pipe(
-        concatMap(jsonResp => {
-            if((jsonResp.dashboardAvailable == true) && (jsonResp.datasourceAvailable == true)) 
-              {return forkJoin([getDatasourceURL, getDashboardURL])}
-            else {this.logError('Grafana configuration', 'URLs unavailable');
-                  return jsonResp}}))
+        concatMap(resp => from(resp.json())), 
+        concatMap((jsonResp: any) => {
+          if ((jsonResp.dashboardAvailable == true) && (jsonResp.datasourceAvailable == true)) {
+            return forkJoin([getDatasourceURL, getDashboardURL])
+          } else {
+              return throwError(new Error('ULRs unavailable'));
+          }}))
       .subscribe(
         (url: any) => 
           {this.grafanaDatasourceUrlSubject.next(url[0].grafanaDatasourceUrl);
-            this.grafanaDashboardUrlSubject.next(url[1].grafanaDashboardUrl);}
+            this.grafanaDashboardUrlSubject.next(url[1].grafanaDashboardUrl);},
```

Comment:
Seems like this was pre-existing, but I think this handler should also be formatted with some newlines.